docs/config.schema.json

```diff
@@ -656,28 +656,16 @@
       "description": "The secrets section configures secrets used for encryption and signing of several systems. All secrets can be rotated, for more information on this topic go to: https://www.ory.sh/docs/hydra/advanced#rotation-of-hmac-token-signing-and-database-and-cookie-encryption-keys",
       "properties": {
         "system": {
-          "type": "array",
           "description": "The system secret must be at least 16 characters long. If none is provided, one will be generated. They key is used to encrypt sensitive data using AES-GCM (256 bit) and validate HMAC signatures. The first item in the list is used for signing and encryption. The whole list is used for verifying signatures and decryption.",
-          "items": {
-            "type": "string",
-            "minLength": 16
-          },
-          "minItems": 1,
-          "examples": [
-            [
-              "this-is-the-primary-secret",
-              "this-is-an-old-secret",
-              "this-is-another-old-secret"
-            ]
-          ]
-        },
-        "cookie": {
+          "default": [
+            "I AM NOT SECRET PLEASE REPLACE ME!!!"
+          ],
           "type": "array",
-          "description": "A secret that is used to encrypt cookie sessions. Defaults to secrets.system. It is recommended to use\ta separate secret in production. The first item in the list is used for signing and encryption. The whole list is used for verifying signatures and decryption.",
           "items": {
             "type": "string",
             "minLength": 16
           },
+          "minItems": 1,
```

Comment:
I think hydra auto-generates a secret when none is set. This has caused some confusion where people forgot to set the system secret, restarted the service, and then wondered why the db connectivity did not longer work.

I am actually not sure if we still generste the secret per default if none is set or not (but I think we do). In that (we do) just set minItems to 0 and say in the description that it is generated per default - or change (and document) the behavior and require a system secret to be set.

Reply:
yes it is generated and there is also log output

Reply:
Ok, in that case `minItems` should be removed and the default as well